tox.ini

```diff
@@ -1,15 +1,13 @@
 [tox]
-envlist = py27, flake8
+envlist = py37, py38, py39, py310, py311, flake8
```

Comment:
Do we want to keep py37?

Reply:
Yes, this was waiting on ability to drop 3.7. Do we want to keep tox?  It won't currently work, and it doesn't actually support 3.10 atm afaics, let alone 3.11. But I don't want to break the tooling people currently use, and I can move the testing logic over to tox if you'd rather.  Else I think we should remove it.  Tox is admittedly more 'pythonic' than a makefile.

In any case the github actions needs to run one python version per matrix run, so they show up as different runs.

Reply:
I've never used tox so it must've been introduced by the previous maintainer before I took over. If it does not work, I suggest we remove it altogether.

Reply:
Right; I'll remove it them.  TBF tox is good at what it does: I just find makefiles more natural, and since I don't keep dozens of pythons around on my machine I only ever test on old versions in github actions, so that's a more natural fit for me.  If we had loads of windows devs I guess we'd need to do tox.